client/components/tinymce/plugins/media/commands.js

```diff
@@ -0,0 +1,14 @@
+export function insertMediaAsLink( editor, media ) {
```

Comment:
In general, Calypso uses a `utils.js` naming convention for files similar to these.